pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1009,5 +1011,41 @@ private static double[] AreaGraphDotProducts
                 return dotpActuals;
             }
         }
+
+        //Tests the redundant spectrum dropdown menu in the ViewSpectralLibraries dialogue

```

Comment:
Use a C# header comment more like this: 

/// &lt;summary>
/// Tests the redundant spectrum dropdown menu in the &lt;see ref="ViewLibraryDlg" /> form
/// &lt;/summary>